### PR TITLE
Update algolia search docs with vendors

### PR DESCRIPTION
**Pull request by sli1989**
## PR Checklist
**Please check if your PR fulfills the following requirements:**

- [x] The commit message follows [our guidelines](https://github.com/theme-next/hexo-theme-next/blob/master/.github/CONTRIBUTING.md).
- [ ] Tests for the changes have been added (for bug fixes / features).
   - [x] Muse | Mist have been tested.
   - [ ] Pisces | Gemini have been tested.
- [x] Docs have been added / updated (for bug fixes / features).

## PR Type
**What kind of change does this PR introduce?**  

- [ ] Bugfix.
- [ ] Feature.
- [ ] Code style update (formatting, local variables).
- [ ] Refactoring (no functional changes, no api changes).
- [ ] Build related changes.
- [ ] CI related changes.
- [x] Documentation content changes.
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number(s): N/A

## What is the new behavior?

Add 3rd module installation instructions and CDN vendors support,

* Screens with this changes: N/A
* Link to demo site with this changes: N/A

### How to use?
In NexT `_config.yml`:
```yml
...
```

## Does this PR introduce a breaking change?
- [ ] Yes.
- [x] No.

### CODE REVIEW

**Comment by sli1989** [comment]
Need more fix. I am working on it.

**Comment by sli1989** [comment]
how to remove 

```
<svg xmlns="http://www.w3.org/2000/svg" id="sbx-icon-search-13" viewBox="0 0 40 40" width="100%" height="100%">
    <path d="M26.804 29.01c-2.832 2.34-6.465 3.746-10.426 3.746C7.333 32.756 0 25.424 0 16.378 0 7.333 7.333 0 16.378 0c9.046 0 16.378 7.333 16.378 16.378 0 3.96-1.406 7.594-3.746 10.426l10.534 10.534c.607.607.61 1.59-.004 2.202-.61.61-1.597.61-2.202.004L26.804 29.01zm-10.426.627c7.323 0 13.26-5.936 13.26-13.26 0-7.32-5.937-13.257-13.26-13.257C9.056 3.12 3.12 9.056 3.12 16.378c0 7.323 5.936 13.26 13.258 13.26z" fill-rule="evenodd">
    </path>
  </svg>

<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 20" width="100%" height="100%">
    <path d="M8.114 10L.944 2.83 0 1.885 1.886 0l.943.943L10 8.113l7.17-7.17.944-.943L20 1.886l-.943.943-7.17 7.17 7.17 7.17.943.944L18.114 20l-.943-.943-7.17-7.17-7.17 7.17-.944.943L0 18.114l.943-.943L8.113 10z" fill-rule="evenodd">
    </path>
  </svg>
```

**Comment by ivan-nginx** [comment]
So, mb in pull this changes?

**Comment by sli1989** [comment]
Fixed already.  Review required.

File: docs/ALGOLIA-SEARCH.md [hunk 1]
```diff
@@ -46,6 +46,32 @@ NexT provides search plugin for index your hexo website content. To use this fea
...
+      ...
+      # Internal version: 1
+      # https://www.algolia.com
+      algolia_instant_js: https://cdn.jsdelivr.net/npm/instantsearch.js@2.4.1/dist/instantsearch.js
+      algolia_instant_css: https://cdn.jsdelivr.net/npm/instantsearch.js@2.4.1/dist/instantsearch.min.css
+      ...
+    ```
+
+    ```
+    // Custom styles.
```
Comment (sli1989) [inline docs/ALGOLIA-SEARCH.md]:
```
.ais-search-box--magnifier svg {
    display: none !important;
}
```

It seems they are all working. But I don't know how to choose one.

Comment (sli1989) [reply docs/ALGOLIA-SEARCH.md]: Confirmed. The PR is right.

Comment (ivan-nginx) [reply docs/ALGOLIA-SEARCH.md]: @sli1989 so, maybe u can add this to [lib plugin](https://github.com/theme-next/theme-next-algolia-instant-search)?

Comment (ivan-nginx) [reply docs/ALGOLIA-SEARCH.md]: Oh no, wait. This need just add to styles in NexT i think.

Comment (sli1989) [reply docs/ALGOLIA-SEARCH.md]: ok, working on it.

Comment (sli1989) [reply docs/ALGOLIA-SEARCH.md]: emm, i will try it.

**Comment by sli1989** [comment]
FIXED in `third-party/algolia-search.styl`.

**Pull request merged**